paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-fr', '--from', dest='time_from',
+        help='The ISO-8601 timestamp to get the logs from. Alternatively, for convenience, time differences can be '
+             'specified in a human readable way, such as --from 5m will grab logs from 5 minutes ago',
+    )
+    status_parser.add_argument(
+        '-t', '--to', dest='time_to',
+        help='The ISO-8601 timestamp to get logs up to. Follows same convention as --from for human readable '
+             'differences. Defaults to right now.'
+    )
+    status_parser.add_argument(
+        '-l', '--lines', dest='line_count',
+        help='The number of lines to retrieve from the specified offset. May optionally be prefixed with a "+" or "-" '
+             'to specify which direction from the offset, defaults to "-100"',
+        type=int,
+    )
+    status_parser.add_argument(
+        '-lo', '--line-offset', dest='line_offset',
```

Comment:
As above, let's stick to the convention that single - arguments have single-character names.